packages/shared/src/lib/localLanguage.ts

```diff
@@ -49,25 +23,18 @@ const localLanguages: LocalLanguages = {
     },
 };
 
-export const countryCodeToVerfiedLocalLanguage = (
+export const countryCodeToLocalLanguageBannerHeader = (
     testName: string,
     variantName: string,
     countryCode?: string,
```

Comment:
It doesn't look like the country code is optional either? If it happens to be in the router, I think there is a `requiredCountry` var that could be used here instead

Reply:
Yes, not optional  now, much clearer and updated tests. Used `requiredCountry` to be sure.